### PR TITLE
BaSP-MR-240: Modify slots

### PR DESCRIPTION
**Pull request by FrancoLelli**
I modify the logic of the slots in the grid, so that when they join a class, the amount available is discounted.
- [X] Modify thunk.
- [X] Modify toogleFunction.
- [X] Add ModalConfirm.

### CODE REVIEW

**Comment by FrancoLelli** [comment]
[Modify "Slots"](https://trello.com/c/iC10lmPM/240-modify-slots)

File: src/redux/classes/thunks.js [hunk 1]
```diff
@@ -74,16 +74,23 @@ export const createClass = async (body, dispatch) => {
 export const updateClass = async (id, body, dispatch) => {
   try {
     dispatch(editClassPending(true));
-    const response = await fetch(`${process.env.REACT_APP_API_URL}/api/class/${id}`, body);
+    console.log(body);
```
Comment (YhanahiRosenthal) [inline src/redux/classes/thunks.js:77]: you should delete that

**Review by CristianLotorto — APPROVED** [review]
Great work!

**Review by YhanahiRosenthal — APPROVED** [review]
Good job!

**Pull request merged**